api/managementpb/actions.proto

```diff
@@ -289,15 +289,13 @@ message StartPTSummaryActionResponse {
   string pmm_agent_id = 2;
 }
 
-// Message to prepare pt-mysql-summary data
```

Comment:
To avoid generating invalid "title" fields in Swagger spec (and, as a result, in UI)